scripts/deploy.sh

```
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+source $(dirname "$0")/environment.sh
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)